src/app/globals.css

```
@@ -35,4 +35,68 @@ body {
+  }
+
+  .animate-marquee-left {
+    animation: marquee-left 20s linear infinite;
+  }
+
+  .animate-marquee-right {
+    animation: marquee-right 20s linear infinite;
+  }
+}
+
+@import url('https://fonts.googleapis.com/css2?family=Unbounded:wght@800&display=swap');
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)